src/utils.ts

```diff
@@ -73,6 +85,15 @@ export function isUrl(input: string): boolean {
   return /^https?:\/\//.test(input);
 }
 
+// Check whether the input is a DID
+export function isDid(input: string): boolean {
+  return /^did:[a-zA-Z0-9]+:[a-zA-Z0-9]+$/.test(input);
+}
+
+export function isIPFSCID(input: string): boolean {
+  return /^ipfs:\/\/[a-zA-Z0.9]+$/.test(input);
+}
```

Comment:
Hmm yeah you're right, this util function was meant for a later commit.. I was looking for a way to validate the ipfs CID, but since the IDX basicProfile already validates it, eventually it is not even necessary.
Will eventually remove this fn.

Reply:
Ah well up to you.